sdbuild/packages/bootpy/boot.py

```diff
@@ -1,30 +1,25 @@
 #! /usr/bin/env python3.6
 
 # example #1 - WiFi connection at boot
-"""
-from pynq.lib import Wifi
+#from pynq.lib import Wifi
+#
+#port = Wifi()
+#port.connect('your_ssid', 'your_password', auto=True)
 
-port = Wifi()
-port.connect('your_ssid', 'your_password', auto=True)
-"""
-# example #2 - Change hostname
```

Comment:
I would like to remove example 2, since this one will keep changing hostname *every* time we boot the board. It should be done just once.